app/views/admin/users/index.html.haml

```diff
@@ -23,10 +23,9 @@
             %td
               = user.id
             %td
-              - if user.confirmed?
-                confirmed
-              - else
-                unconfirmed
+              - if can? :update, user
+                = check_box_tag user.id, user.id, user.confirmed?, method: :patch, url: "/admin/users/#{user.id}/toggle_confirmation?value=", class: 'switch-checkbox', data: { size: 'small', off_color: 'warning', on_text: 'Yes', off_text: 'No' }
+
```

Comment:
Okay, and also can you please explain what the three parameters next to the check_box_tag means. I am having a hard time figuring that out.